src/helix-ui/styles/core/hx-checkbox.less

```diff
@@ -1,8 +1,9 @@
 hx-checkbox {
   display: inline-block;
   height: 1rem;
+  vertical-align: text-top;
```

Comment:
This will align the bottom of the checkbox with the baseline of surrounding text.